acra/configuring-maintaining/metrics/_index.md

```diff
@@ -1,4 +1,75 @@
 ---
+
+weight: 1
 title: Metrics
 bookCollapseSection: true
 ---
+
+# Metrics in Acra
+
+Starting with version [`0.83.0`](https://github.com/cossacklabs/acra/releases/tag/0.83.0), Acra supports tracking and exporting of the basic metrics of AcraServer, AcraConnector, and AcraTranslator to [` Prometheus`](https://prometheus.io/).
+
+
+## Metrics configuration
+
+### Command line flags
+
+* `--incoming_connection_prometheus_metrics_string=<url>`
+
+  URL (tcp://host:port) which will be used to expose Prometheus metrics (`<URL>/metrics` address to pull metrics).
+
+
+### Exposed metrics
+
+Acra can expose standard set of metric [`types`](https://prometheus.io/docs/concepts/metric_types/) for AcraServer, AcraConnector, and AcraTranslator. 
+Additional metrics registration info could be found in [`prometheus.go`](https://github.com/cossacklabs/acra-Q12021/blob/master/cmd/acra-server/common/prometheus.go) files located in `cmd` folder for each corresponded Acra component.
+
+
+
+|                            Metric                                   |              Description                   |                                     Type                                   | 
+| ------------------------------------------------------------------- | ------------------------------------------ | -------------------------------------------------------------------------- | 
+|  `acra{server|translator|connector}_connections_total`              | Number of connections to database          | [`Counter`](https://prometheus.io/docs/concepts/metric_types/#counter)     | 
+|  `acra{server|translator|connector}_connections_processing_seconds` | Time of connection processing              | [`Histogram`](https://prometheus.io/docs/concepts/metric_types/#histogram) |
+|  `acra{server|translator}_request_processing_seconds`               | Time of request processing                 | [`Histogram`](https://prometheus.io/docs/concepts/metric_types/#histogram) |
+|  `acraserver_response_processing_seconds`                           | Time of response processing                | [`Histogram`](https://prometheus.io/docs/concepts/metric_types/#histogram) |
+|  `acra_api_encryptions_total`                                       | Number of encryptions data to AcraStruct   | [`Counter`](https://prometheus.io/docs/concepts/metric_types/#counter)     |
```

Comment:
can we describe labels too? all our labels are declared as constants, not dynamic. for example `fail/success` labels for acrastruct descriptions. plus set of buckets configured for histograms may be useful.

Reply:
Yes, we can add this too but current table have some side scroll because the content is too big. Do we really want to do more columns here. This is how it looks like. So in my opinion it is better to just present some genral info and if you need to get more detailed info you have reference in code. Or what do you think about that? @Lagovas 
![Screenshot from 2021-09-10 12-21-16](https://user-images.githubusercontent.com/24659475/132832392-237a3fc5-37c4-433c-b82a-b33153e63e55.png)

Reply:
yes. because main goal of documentation is to help users understand how it works without reading code)